lib/common/api/deprecate.ts

```diff
@@ -77,57 +75,57 @@ const deprecate = {
     })
   },
 
-  promisify: (fn) => {
+  promisify: <T extends (...args: any) => any>(fn: T): T => {
```

Comment:
```suggestion
  promisify: <T extends (...args: any[]) => any>(fn: T): T => {
```
right?

Reply:
Yeah, interestingly `any[]` is a subset of `any`, updating 👍